wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -351,6 +355,17 @@ impl PhysicalDeviceFeatures {
             } else {
                 None
             },
+            shader_int64_atomic: if device_api_version >= vk::API_VERSION_1_1
```

Comment:
This seems to be checking if image atomics are supported, not if regular 64bit atomics are supported.